test/context.test.js

```diff
@@ -61,7 +62,7 @@ describe('context', () => {
     });
 
     it('should return mockOrderRecipientFactory.create if piiServerUrl is present', () => {
-      config.set('piiServerUrl', faker.internet.url());
+      config.init(mockConfig({ piiServerUrl: faker.internet.url() }));
```

Comment:
Isn't `piiServerUrl: faker.internet.url()` already set in `mockConfig`? This is a tiny example of decoupling troubles, and another basis for using two mock configs, I'd say

Reply:
yes, changing `mockConfig({ piiServerUrl: faker.internet.url() })` to `mockConfig()` should not break this test. However, given your previous [comment](https://github.com/rewardops/rewardops-sdk-node/pull/46#discussion_r501051125) I think we should mock `config` so that tests are atomic.

Reply:
> we should mock config so that tests are atomic.

i.e. mock a standard as well as a PII config?

Reply:
that sounds good